.travis.yml

```diff
@@ -2,7 +2,9 @@ sudo: false
 
 language: cpp
 
+# apt seems to be broken in bionic, so we need a workaround
```

Comment:
Have you seen some bug report about this? If so, please include it in this comment and in the git commit case. If not, please report it.

Would it make sense to skip bionic for now, rather than add this workaround?

Reply:
No, I haven't found any official reports about it, but the logs are pretty clear. If you look at https://travis-ci.org/ricrogz/maeparser/jobs/568401552 (one of the builds from this morning, when I was playing with this in my fork of maeparser), instead of the part of the log where the automatic `apt update` and the installation of packages listed under `addons > apt > packages` happens, you see this (lines 167-169):
```
sudo: unable to resolve host travis-job-6662e041-cc8b-426f-9269-c523e095877c
sudo: unable to resolve host travis-job-6662e041-cc8b-426f-9269-c523e095877c
sudo: unable to resolve host travis-job-6662e041-cc8b-426f-9269-c523e095877c
```
I also googled around a bit, and noticed it also happens in other git repos (sorry, I no longer have the specific URLs I looked at).

The workaround is very simple: instead of relying on the automated apt, just install these packages ourselves. The only tricky part is getting it to happen in Bionic, and not in the other builds, and that is still easy (took me a couple of tries, though). Also, the workaround would not cause any kind of trouble if this gets fixed in the future: the packages would still get installed manually, instead of automatically.

I don't think we should skip Bionic. I think it makes total sense to have a build for it, since it is the latest LTS distribution available, and there's probably a good number of people that are using it.